### PR TITLE
[WIP] Member profile page

### PR DESCRIPTION
**Pull request by whyDontI**

### CODE REVIEW

File: components/member-profile/index.js [hunk 1]
```diff
@@ -8,10 +8,157 @@ const Profile = (props) => {
...
+          <div className={classNames.socialIcons}>
+            <img
+              src="https://staging-members-rds.herokuapp.com/icons/instagram.png"
+              alt="Instagram"
+            />
+            <img
+              src="https://staging-members-rds.herokuapp.com/icons/linkedin.png"
+              alt="LinkedIn"
+            />
+            <img src="https://staging-members-rds.herokuapp.com/icons/github.png" alt="GitHub" />
```
Comment (sumitd94) [inline components/member-profile/index.js:36]: NIT : the src links , we can store this in a const variables at the top and then use it here
example : `Instagram`
The same goes with LinkedIn, Github , Twitter, etc

Comment (whyDontI) [reply components/member-profile/index.js:36]: Sure, Will make the changes.

Comment (whyDontI) [reply components/member-profile/index.js]: @sumitd94 Changes are done, can you please approve?

**Review by sumitd94 — APPROVED** [review]

File: styles/layout-style.module.scss [hunk 2]
```diff
@@ -4,7 +4,3 @@
   width: 100%;
   padding-bottom: 20px;
 }
-
-.main {
-  margin: 20px;
-}
```
Comment (whyDontI) [inline styles/layout-style.module.scss:10]: Removed this margin to remove horizontal scrolling

**Review by ankushdharkar — APPROVED** [review]

**Pull request merged**